wdqs/runUpdate.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# This file is provided by the wikibase/wdqs docker image.
+
+cd /wdqs || exit
+
+# TODO env vars for entity namespaces, scheme and other settings
+/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
```

Comment:
why do we need the port here

Reply:
Well, wait-for-it.sh expects a port and it fails otherwise. But this port 80 was already hard-coded in the original runUpdate.sh from wdqs. I didn't touch this line.

All I did was add the port also in line 9 to avoid the errors we were getting such as:
"org.wikidata.query.rdf.tool.rdf.Munger$BadSubjectException: Unrecognized subjects:  [http://localhost/entity/Q1, http://wikibase.svc:80/wiki/Special:EntityData/Q1].  Expected only sitelinks and subjects starting with http://wikibase.svc/wiki/Special:EntityData/ and [http://wikibase.svc/entity/]"

It could still be nicer to substitute it with a variable in a next step. But I would test first if synchronization works this way (it does at least in local development).